src/plotting/task_plot_rapid_test_statistics.py

```diff
@@ -0,0 +1,120 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import pytask
+import seaborn as sns
+
+from src.config import BLD
+from src.config import PLOT_SIZE
+from src.plotting.plotting import BLUE
+from src.plotting.plotting import RED
+from src.plotting.plotting import style_plot
+from src.simulation.scenario_config import create_path_to_rapid_test_statistics
+from src.simulation.scenario_config import get_named_scenarios
+
+SEEDS = get_named_scenarios()["combined_baseline"]["n_seeds"]
+
+_DEPENDENCIES = {
+    seed: create_path_to_rapid_test_statistics("combined_baseline", seed)
+    for seed in range(SEEDS)
+}
+
+
+@pytask.mark.skipif(SEEDS == 0, reason="combined_baseline did not run.")
+@pytask.mark.depends_on(_DEPENDENCIES)
+@pytask.mark.produces(
+    {
+        "demand_shares": BLD
+        / "figures"
+        / "rapid_test_statistics"
+        / "demand_shares.pdf",
+        "share_infected_among_demand": BLD
+        / "figures"
+        / "rapid_test_statistics"
+        / "share_infected_among_demand.pdf",
+        "demand_shares_with_single_runs": BLD
+        / "figures"
+        / "rapid_test_statistics"
+        / "demand_shares_with_single_runs.pdf",
+        "share_infected_among_demand_with_single_runs": BLD
+        / "figures"
+        / "rapid_test_statistics"
+        / "share_infected_among_demand_with_single_runs.pdf",
+    }
+)
+def task_plot_rapid_test_statistics(depends_on, produces):
+    dfs = {
+        seed: pd.read_csv(path, parse_dates=["date"], index_col="date")
+        for seed, path in depends_on.items()
+    }
+    for df in dfs.values():
+        assert not df.index.duplicated().any(), (
```

Comment:
I think this has the potential to be really annoying. Can't we delete those files automatically at the start of each run?

Reply:
Pytask does this (see `task_run_simulation.py:40-43`). I decided to check in addition.